libqtile/core/base.py

```diff
@@ -1,8 +1,27 @@
 from abc import ABCMeta, abstractmethod
+
 import typing
+import asyncio
+
+from . import xcbq
+
+
+class ShutdownEvent:
+    message: str
```

Comment:
I think this should be removed, I think this defines a class variable `ShutdownEvent.message`, which we don't really want. I feel like this is to appease the mypy linter, if that's the case, you could maybe try the `NamedTuple` in `typing`?

Reply:
Sounds good!